ENGINE.md

```diff
@@ -1,10 +1,11 @@
 # Database Engine Support
 
 ## v6
-| Engine | Minimum supported version |
-| :------------: | :------------: |
-| PostgreSQL | [9.5](https://www.postgresql.org/docs/9.5/ ) |
-| MySQL | [5.7](https://dev.mysql.com/doc/refman/5.7/en/) |
-| MariaDB | [10.3](https://mariadb.com/kb/en/changes-improvements-in-mariadb-103/) |
-| Microsoft SQL Server | `12.0.2000` |
-| SQLite | [3.0](https://www.sqlite.org/version3.html) |
```

Comment:
I don't get where this version number is coming from. I would rather say that the minimum supported version is 3.0 and that Sequelize defaults to SQLite 3.32.3 in case the system doesn't already have it available: https://github.com/mapbox/node-sqlite3#features

```
Bundles SQLite3 3.32.3 as a fallback if the installing system doesn't include SQLite

```

Reply:
> I don't get where this version number is coming from. I would rather say that the minimum supported version is 3.0 and that Sequelize defaults to SQLite 3.32.3 in case the system doesn't already have it available: https://github.com/mapbox/node-sqlite3#features
> 
> ```
> Bundles SQLite3 3.32.3 as a fallback if the installing system doesn't include SQLite
> ```

The version number is coming from `lib/dialects` to tell the Sequelize users if the database is not supported anymore. You can see the deprecated message in `lib/utils/deprecations.js`

Reply:
postgres `9.5.0` is my mistake, i thought the dialects documentations specify the supported version as well.

Reply:
Ah you are referring to the defaultVersion. OK. Why not :D